### PR TITLE
Fix link format

### PR DESCRIPTION
**Pull request by meker12**
## Purpose of this pull request

Fixes an incorrect link format.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.4/config-guide/prod/config-reference-var-name.html#how-to-use-environment-variables

### CODE REVIEW

File: src/guides/v2.3/config-guide/prod/config-reference-var-name.md [hunk 1]
```diff
@@ -242,7 +242,7 @@ content='
 
 *  To use values you set in the `$_ENV` array, you must set `variables_order = "EGPCS"` in your `php.ini` file. For details, see [PHP documentation](http://us.php.net/manual/en/ini.core.php#ini.variables-order).
 
-*  For Magento Commerce Cloud, if you are attempting to override Magento configuration settings using the [Project Web Interface]((https://devdocs.magento.com/guides/v2.3/cloud/project/project-webint-basic.html#project-conf-env-var), you must prepend the variable name with `env:`. For example:
+*  For Magento Commerce Cloud, if you are attempting to override Magento configuration settings using the [Project Web Interface](https://devdocs.magento.com/guides/v2.3/cloud/project/project-webint-basic.html#project-conf-env-var), you must prepend the variable name with `env:`. For example:
```
Comment (BarnyShergold) [inline src/guides/v2.3/config-guide/prod/config-reference-var-name.md:245]: Whilst this is being changed, perhaps change the link to the new one  - https://devdocs.magento.com/cloud/project/project-webint-basic.html#project-conf-env-var

**Review by BarnyShergold — CHANGES_REQUESTED** [review]

**Comment by dobooth** [comment]
Hardcoded link to v2.3?

File: src/guides/v2.3/config-guide/prod/config-reference-var-name.md (hunk 1 above)
Comment (meker12) [inline src/guides/v2.3/config-guide/prod/config-reference-var-name.md:245]:
```suggestion
*  For Magento Commerce Cloud, if you are attempting to override Magento configuration settings using the [Project Web Interface]({{ site.baseurl }}/cloud/project/project-webint-basic.html#project-conf-env-var), you must prepend the variable name with `env:`. For example:
```

**Review by BarnyShergold — APPROVED** [review]

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
running tests

**Pull request merged**